src/pages/public_registration/PublicProjectsContent.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Chip, MuiThemeProvider, Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import { PublicRegistrationContentProps } from './PublicRegistrationContent';
+import lightTheme from '../../themes/lightTheme';
+
+const StyledProjectChip = styled(Chip)`
+  margin: 0.25rem;
+  background: #ff8888;
+`;
+
+const PublicProjectsContent = ({ registration }: PublicRegistrationContentProps) => {
+  const { t } = useTranslation('registration');
+
+  return (
+    <>
+      <Typography variant="h4" component="h2" gutterBottom>
+        {t('registration:description.project_association')}
+      </Typography>
+
```

Comment:
Alt under her må nok skrives om når Projects får mer data, så har bare fått det til å matche skisser for nå